### PR TITLE
Update CI

### LINKED ISSUE #136: Mambaforge is getting sunsetted
We should remove it from the template

### PR DESCRIPTION
**Pull request by lilyminium**
Fixes #136 

Changes made in this Pull Request:
 - Updates to miniforge/miniconda instead of mambaforge

PR Checklist
------------
 - [ ] Tests?
 - [ ] Docs?
 - [ ] CHANGELOG.md updated?
 - [ ] AUTHORS.md updated?
 - [x] Issue raised/referenced?

### CODE REVIEW

File: .github/workflows/gh-ci.yaml [hunk 1]
```diff
@@ -158,7 +158,7 @@ jobs:
           environment-file: docs/requirements.yaml
           add-pip-as-python-dependency: true
           architecture: x64
-          miniforge-variant: Mambaforge
+          miniforge-version: "latest"
           use-mamba: true
           channels: conda-forge, defaults
```
Comment (IAlibay) [inline .github/workflows/gh-ci.yaml:163]: Can we remove `defaults`? It's problematic with the whole anaconda license stuff.

File: {{cookiecutter.repo_name}}/.github/workflows/{{cookiecutter._ci_name}}.yaml [hunk 2]
```diff
@@ -66,10 +66,11 @@ jobs:
         environment-file: devtools/conda-envs/test_env.yaml
         add-pip-as-python-dependency: true
 {% if cookiecutter.__dependency_source == 'conda-forge' %}
-        miniforge-variant: Mambaforge
+        miniforge-version: "latest"
         use-mamba: true
         channels: conda-forge, defaults
```
Comment (IAlibay) [inline {{cookiecutter.repo_name}}/.github/workflows/{{cookiecutter._ci_name}}.yaml:71]: Same things here re: defaults

**Review by IAlibay — APPROVED** [review]
This looks good to me, but we should remove `defaults` from the channels.

There are issues in the MDA mdakits that are failing even with these changes though - so we should work out what's up with them (i.e. there might be a deeper issue).

File: {{cookiecutter.repo_name}}/.github/workflows/{{cookiecutter._ci_name}}.yaml (hunk 2 above)
Comment (lilyminium) [reply {{cookiecutter.repo_name}}/.github/workflows/{{cookiecutter._ci_name}}.yaml]: Done, and raised #142 to tackle environment files.

**Comment by lilyminium** [comment]
Thanks for the review @IAlibay, once CI goes goes green (fingers crossed) I'll merge.

**Pull request merged**